pyomo/core/expr/current.py

```diff
@@ -154,3 +162,5 @@
     identify_variables,
     identify_mutable_parameters,
 )
+
+_mode = Mode.CURRENT
```

Comment:
Could you add a short comment explaining this line?

Reply:
Actually, this should be (and has been) removed.  The correct definition is imported from `expr_common` above.